src/modules/bricks/volume-bricks/volume-bricks.js

```diff
@@ -310,6 +335,17 @@
                 match = brick.status === filter.value.id || brick.status.toLowerCase() === filter.value.toLowerCase();
             } else if (filter.id === "devices") {
                 match = brick.devices[0].match(re) !== null;
+            } else if (filter.id === "utilMoreThan" || filter.id === "utilLessThan") {
+                percentage = parseInt(filter.value);
+                utilization = parseInt(brick.utilization.used);
+
+                if (percentage < 0 || percentage > 100) {
+                    vm.errorMsg = "Please enter a valid percentage.";
+                } else if (filter.id === "utilMoreThan") {
+                    match = (utilization > percentage);
+                } else if (filter.id === "utilLessThan") {
+                    match = (utilization < percentage);
+                }
```

Comment:
@gnehapk condition when both utilization and percentage are same should also be added

Reply:
@a2batic Since, the filter title is `more than/less than`, so ideally it should filter bricks whose utilization is more than/less than the entered value. So, if the bricks utilization is 34%, then user needs to enter any value less than this in `Utilization More Than` filter to filter it.

Reply:
For example if percentage entered by user is 50, and the brick utilization is also 50, it wont be displayed in either `utilization more` or `utilization less`. @julienlim @mcarrano, where will the equal values be filtered, less than or more than ?

Reply:
@a2batic @mcarrano Perhaps we should then do utilization less than or equal.

Thoughts?

Reply:
I don't have a strong opinion about this. We can change the "less than" to "less than or equal to" as you suggest @julienlim . I guess the question is does it matter? If as a user I wanted to see bricks with a utilization of less than 50% or more than 50%, then do I care if I don't see the ones that are exactly 50%?